js/menu.js

```diff
@@ -0,0 +1,12 @@
+const hamburguer = document.querySelector('.hamburguer');
+const navMenu = document.querySelector('.nav-menu');
+
+hamburguer.addEventListener('click', () => {
+  hamburguer.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
```

Comment:
Kindly make sure the functionality is working as when I click the hamberger button the navigation menu doesn't appear , and I couldn't find the active class in the css file please do your best to make it work and if you need any help tag me in the comments

Reply:
@souad988 Are you sure you have cloned all the files? Because the hamburger menu works for me, it displays the nav and the close button works.
![Captura de pantalla (93)](https://github.com/maximoortelli/Portfolio-setup-and-mobile--first/assets/118080546/1b93b7b1-5349-49d9-98bb-38e10d00564d)
![Captura de pantalla (94)](https://github.com/maximoortelli/Portfolio-setup-and-mobile--first/assets/118080546/92cf12ef-83bb-4f60-9f90-35ed7fc7b125)

Reply:
![Captura de pantalla (99)](https://github.com/maximoortelli/Portfolio-setup-and-mobile--first/assets/118080546/5bd669e2-8fcf-4e61-8d23-013f4db9fa92)
The code that you have marked is connected with my CSS working correctly, and with the Hamburger class that also belongs to my index and is in CSS.

Reply:
I would like if you could give an example of how to solve it optimally or be more specific or clear.